src/modules/auth/components/SignUp/FormSignUp.component.tsx

```diff
@@ -3,19 +3,21 @@ import { Button } from '@shared/components/Button/Button.component';
 import { useTranslation } from 'react-i18next';
 import { useForm } from 'react-hook-form';
 import { Checkbox } from '../../../../shared/components/Checkbox/Checkbox.component';
+import { useAppDispatch } from '@store/index.store.ts';
+import { trainerSignupAction } from '@modules/auth/store/authActions.store.ts';
 
 export interface FormInputs {
   firstName: string;
   lastName: string;
   terms: boolean;
-  date: string;
+  birthdate: string;
```

Comment:
IS this a typo? 
Should we call it `birthday`? 🤔

Reply:
Backend variable name, will change it there also